types/googlemaps/index.d.ts

```diff
@@ -42,6 +42,12 @@ THE SOFTWARE.
 */
 
 declare namespace google.maps {
+    /**
+     * Version of the Google Maps JavaScript API that the browser has loaded.
+     * Like '3.38.11'.
```

Comment:
I propose to add a link to the source.

```suggestion
     * Like '3.38.11'.
     * @see {@link https://developers.google.com/maps/documentation/javascript/versions#version-checks Maps JavaScript API}
```